.circleci/config.yml

```
@@ -9,10 +9,23 @@ jobs:
           keys:
             - v1-dependencies-{{ checksum "package.json" }}
             - v1-dependencies-
-      - run: yarn install
+      - run:
+          name: install dependencies
+          command: yarn install
       - save_cache:
           paths:
             - node_modules
           key: v1-dependencies-{{ checksum "package.json" }}
-      - run: yarn lint
-      - run: yarn test
+      - run:
+          name: lint
+          command: yarn lint
+      - run:
+          name: test
+          command: yarn test
+      - run:
+          name: install apollo-client example dependencies
+          command: cd examples/apollo-client && yarn install
```

Instant of cd is possible also to set the `working_directory` in this way:
```suggestion
          working_directory: examples/apollo-client
          command: yarn install
```
also, it would be possible to set the working_directory for the entire job but then we will need to create a workflow in order to define the execution order.

If you would like you can give a look to workflows: https://circleci.com/docs/2.0/workflows/